lib/std/zig.zig

```diff
@@ -716,6 +712,11 @@ pub const EnvVar = enum {
     pub fn getPosix(comptime ev: EnvVar) ?[:0]const u8 {
         return std.posix.getenvZ(@tagName(ev));
     }
+
+    pub fn isSet(ev: EnvVar, arena: std.mem.Allocator) !bool {
+        const value = try ev.get(arena) orelse return false;
+        return value.len != 0;
+    }
```

Comment:
It feels a little silly to be allocating just to check if it has a non-zero length. It's not a huge deal given the places this is called in, but it does make me feel like the `std.process` API could use some enhancements in the future.

Reply:
I agree. It would be great if we could have a version of `std.process.getEnvVarOwned` that includes a boolean if an allocation is required.